### DISCUSSION TITLE
Ask Siri to pause the music

### DISCUSSION THREAD

**Discussion by leo150** [mikebrady/shairport-sync#1630] [Q&A]
When I ask Siri on my HomePod to pause the music "Hey Siri, pause music on Berry", it says "Working on it" and nothing happened. Berry is my RP shairport service name. Some other commands like "Hey Siri, play music on Berry" or "Hey Siri, transfer music to Berry" work fine. I can see that HomePod tries to connect to the RP:
```
7.344874625 "rtsp.c:2946" Connection 2: Remote Control connection from [redacted] ("Kitchen") to self at [redacted].
0.069656259 "rtsp.c:1372" Connection 2: Connection closed by client.
```
Is it something intended to work? Thanks.

**Comment by mikebrady** [comment]
Thanks. You'll hate this, but "It works for me!" I am inclined to say "Hey Siri, stop the music in the Kitchen, please." and it almost always works...
